### PR TITLE
Alphabetize fields

### PR DESCRIPTION
**Pull request by damassi**
This PR goes through and alphabetizes all type properties, making it much easier to navigate through our GraphQL schema explorer, build queries and add additional fields without having to dig or feel disoriented. 

I've gone through and compared before and after line counts in the diff, but would appreciate a second set of eyes since this PR contains a lot of detail. Two files differ in length and are noted in comments (I removed some whitespace).

(Whoever thought to enforce trailing commas at the onset, 🙏 🙇 ; it made this reorganization work much easier.)  

schema

### CODE REVIEW

File: schema/artist/index.js [hunk 1]
```diff
@@ -463,20 +459,17 @@ const ArtistType = new GraphQLObjectType({
...
+        type: new GraphQLList(Show.type),
+        ...ShowField,
+      },
+      sortable_id: {
+        type: GraphQLString,
+        description: 'Use this attribute to sort by when sorting a collection of Artists',
+      },
+      statuses: ArtistStatuses,
+      years: {
+        type: GraphQLString,
```
Comment (damassi) [inline schema/artist/index.js]: Removed whitespace, hence the 7-line difference.

File: schema/image/proxies/embedly.js [hunk 2]
```diff
@@ -6,16 +6,15 @@ const {
...
+    crop: {
       url: src,
       width,
       height,
       key: EMBEDLY_KEY,
       quality: 95,
     },
-
-    crop: {
+    resize: {
```
Comment (damassi) [inline schema/image/proxies/embedly.js]: Remove one line of whitespace

**Comment by orta** [comment]
GraphiQL should be doing this ;)

**Comment by alloy** [comment]
@orta makes a good point for the general case, but I’m not opposed to being able to more easily browse the source code.

How has this change been made, by hand? I did a pass and wasn’t able to find any issue, but I don’t feel convinced I might not have missed anything.

In any case, isn’t there an ESLint rule that could be added to ensure future changes follow the same alphabetical sort?

**Comment by damassi** [comment]
@orta - There's some benefit to being able to arrange object keys however you please (things like `width`, `height`come to mind), but once a schema gets to a certain size it makes more sense to treat it like a book index. I think those responsible for GraphiQL wanted to keep it flexible in this way. 

@alloy - Yup, very carefully by hand, and so I understand the caution here. Unfortunately all of the various editor sorting mechanisms fail when it comes to sorting keys with values that tree out. 

As far as automatic enforcement goes, unfortunately no -- http://eslint.org/docs/rules/sort-keys only works for simple object literals. To keep the schema clean we would have to recommend that people try to follow strict alpha (which I think is a good thing to enforce for readability and scale).

**Comment by alloy** [comment]
Alright, I'm +1 on merging, but I'll leave that up to you. I'd suggest a deploy to production before merging this, so this can sit in staging for a little bit.

**Comment by alloy** [comment]
Oh, I guess @broskoski just did that :)

**Comment by broskoski** [comment]
I didn't deploy yet!

**Comment by damassi** [comment]
@alloy, @broskoski - I'll send master to prod and then merge this in; one sec

**Comment by damassi** [comment]
Alright, mergin'

**Comment by broskoski** [comment]
Well done.

**Comment by damassi** [comment]
I'm going to keep digging around the eslint ecosystem for a way to better lock this down, but we should try to figure out how to encourage people to consciously keep alphabetization going as new PR's are added. I've found in the past that most people never think about alpha (in code -- all code -- in general) but then once it's discovered and applied in practice the benefits become obvious. I think our schema was just another extension of that. Now that Metaphysics is of a substantial size we should make an effort to maintain it like an index, especially since it's been opened up to the larger community.

**Pull request merged**